extension/resources/walkthrough/plots.md

```diff
@@ -1,9 +1,17 @@
 # Plots Dashboard
 
-Select one or more experiments to visualize in the
-[**Plots Dashboard**](command:dvc.showPlots). Use
+Select (click the circle beside the experiment name) up to 7 experiments in the
+table view to visualize in the [**Plots Dashboard**](command:dvc.showPlots).
+
+<p align="center">
+  <img src="images/plots-click-the-circle-beside-experiment-name.png"
```

Comment:
This one is still missing: 

<img width="1728" alt="image" src="https://user-images.githubusercontent.com/37993418/196314754-1cc51299-e252-44d8-928c-272db4cbb86e.png">